help/start-a-call.md

```diff
@@ -42,12 +44,39 @@
 
 {end_tabs}
 
-## Change your video call provider
+## Start a voice call
+
+{start_tabs}
+
+{tab|desktop-web}
+
+{!start-composing.md!}
+
+1. Click the **phone** (<i class="fa fa-phone"></i>) icon at the
+   bottom of the compose box. This will insert a **Join voice call.** link into
+   your message.
+
+1. Send the message.
+
+1. Click on the link in the message to start or join the call.
+
+!!! tip ""
+
+    You can replace the "Join voice call." label for the link with any text you
+    like.
+
+{end_tabs}
+
+!!! warn ""
+
+    **Note:** Voice calls are not currently supported for BigBlueButton.
+
```

Comment:
It's actually a bit more complicated; see https://chat.zulip.org/#narrow/stream/101-design/topic/audio-only.20calls/near/1679883.

Let's make the note be regular text after the list of supported call providers

> If you choose BigBlueButton as the call provider, there will be a single button (<img src="/static/images/help/mobile-video-icon.svg" alt="video" class="help-center-icon"/>) for starting a call. The call is initiated with cameras turned off.

@N-Shar-ma , just to confirm, that's the current state in `main`, right?